metricflow/plan_conversion/dataflow_to_sql.py

```diff
@@ -286,23 +296,81 @@ def _make_time_spine_data_set(
                 )
             # TODO: also handle date part.
 
+        output_instance_set = InstanceSet(
```

Comment:
This output set was incorrect before this PR, but was never used for any real logic so it didn't matter.